libs/npm/core/tsconfig.json

```diff
@@ -7,11 +7,10 @@
     "declaration": true,
     "declarationMap": true,
     "emitDeclarationOnly": false,
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "paths": {
+      "@assets/*": ["../../../../app/subscriber/public/assets/*"]
```

Comment:
I don't understand the need for this?  Shouldn't we keep tno-core assets locally?  If it's the shared library then the apps that use the shared library should be getting their components/assets from tno-core.  It is also easier to maintain.

Reply:
Thank you for bringing this, I wanted to use the proper icons instead of applying style:
`<FaSmile className="tone-icon" color="#59E9BE" />`
The icons are under `app/subscriber/public/assets/reports/`, so I created an alias to be used in the import. Then I could use `import @reports/face-negative@2x.png`